docs/synapse/userguides/storm_ref_model_introspect.ipynb

```diff
@@ -251,7 +251,7 @@
     "q = 'syn:prop:type = inet:fqdn'\n",
     "print(q)\n",
     "# Execute the query and test\n",
-    "podes = await core.eval(q, num=48, cmdr=False)"
```

Comment:
Is what this asserts no longer valid?

Reply:
It's a number than changes (and breaks) every time any form gains a secondary property that is an `inet:fqdn` and i'm tired of it breaking :)

Reply:
:+1: